src/app/Table.elm

```diff
@@ -0,0 +1,29 @@
+module Table exposing (view, init, Msg)
```

Comment:
What was I thinking... `tableView` is just a main level view helper function.

Re TEA and nested architecture, I view them as two terms that describe the same thing. The counter example in the old guide showed us how to wire up init, view, update and Msg of a child component such that it held it's own state, did it's own update and rendered it's own view.

Searching for '0.18 TEA nested components elm' gives me https://groups.google.com/forum/#!msg/elm-discuss/dC0htGQOssY/xhHMjWd9EAAJ which discusses mdl and also the nested component movement. There has been alot of talk of it in slack over the last few months since the 0.18 move too.

However, I'm still not clear as to what replaces it, if anything... which is why I'm still exploring using cotwelm. One thing of note is that whilst TEA makes alot of sense in some situations, the vast majority of cotw does not use TEA and is fine.